tests/sca_package/test_output.py

```diff
@@ -176,7 +176,7 @@ def test_create_report_cve_record_results_from_platform():
         check_class=check_class,
         vulnerability_details=vulnerability_details,
         licenses='OSI_BDS',
-        scan_data_format=ScanDataFormat.FROM_PLATFORM
+        scan_data_format=ScanDataFormat.PLATFORM_FORMAT
```

Comment:
it would be great, if you can also add an actual output test or it will come later?

Reply:
I just waiting for Noa to finish the output and I will add tests for that :)